File: dsa-java/ConstantTime.java
public class ConstantTime{
    public static void main(String[] args){
        System.out.println("this line runs in constant time O(1)");

        for(int i = 0; i < 1; i++){
            System.out.println("this line runs in constant time O(1)");
        }

        int x = 5;
        while(x == 5){
            System.out.println("this line runs in constant time O(1)");
            x++;
        }

        int y = 7;
        do{
            System.out.println("this line runs in constant time O(1)");
            y++;
        }
        while(y == 7);
    }
}

File: dsa-cc/constant_time.cc
#include <iostream>

int main() {
    printf("this line runs in constant time O(1) \n");
    
    for(int i = 0; i < 1; i++){
        printf("this line runs in constant time O(1) \n");
    }
    
    int x = 5;
    while(x == 5){
        printf("this line runs in constant time O(1) \n");
        --x;
    }

    int y = 5;
    do{
        printf("this line runs in constant time O(1) \n");
        --y;
    }
    while(y == 5);
    return 0;
}